tensorflow/contrib/igfs/kernels/igfs_client.h

```diff
@@ -0,0 +1,109 @@
+/* Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef TENSORFLOW_CONTRIB_IGFS_KERNELS_IGFS_CLIENT_H_
+#define TENSORFLOW_CONTRIB_IGFS_KERNELS_IGFS_CLIENT_H_
+
+#include "tensorflow/contrib/igfs/kernels/igfs_messages.h"
+
+namespace tensorflow {
+
+class IGFSClient {
+ public:
+  IGFSClient(const string &host, int port, const string &fs_name,
+             const string &user_name);
+  ~IGFSClient();
+
+  inline Status Handshake(CtrlResponse<HandshakeResponse> *res) {
```

Comment:
`inline` is implied for member functions defined in the class definition.

Reply:
Yes, agree. It's exactly member function defined in the class definition.

Reply:
Okay, I got it. I've removed `inline` in headers here and in `ignite_client.h` and `ignite_byte_swapper.h` as well.